src/util/tracking.ts

```diff
@@ -59,22 +56,44 @@ export type TrackingEventName =
 
 export type OnLogEvent = (event: TrackingEventName, values?: TrackingValues) => void
 
+// Known dollar amount revenue
```

Comment:
Can you use `/** */` for these comments? Then VSCode will treat them as documentation and show them when you hover the type name.

Reply:
Thought that only was for fns! This is obvious in hindsight thanks